### PR TITLE
Created AnimePreview repository

### PR DESCRIPTION
**Pull request by turchin-aa**

### CODE REVIEW

File: Core/Enums/VideoTags.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Core.Enums
+{
+	public enum VideoTags
+    {
```
Comment (ShutKuva) [inline Core/Enums/VideoTags.cs:4]: Fix tabs, please

File: Core/DB/AnimePreview.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+namespace Core.DB
+{
+	public class AnimePreview : BaseEntity
+	{
+		public string Title { get; set; } = String.Empty;
+		public string Placement { get; set; } = String.Empty;
+		public string Duration { get; set; } = String.Empty;
+		public string Description { get; set; } = String.Empty;
+		public string ImageUrl { get; set; } = String.Empty;
+        public DateTime Date { get; set; }
```
Comment (ShutKuva) [inline Core/DB/AnimePreview.cs:12]: Fix tabs, please

File: Core/Enums/VideoTags.cs (hunk 1 above)
Comment (turchin-aa) [reply Core/Enums/VideoTags.cs:4]: Tabs are ok. That's how GitHub displays them.

File: Core/DB/AnimePreview.cs (hunk 2 above)
Comment (turchin-aa) [reply Core/DB/AnimePreview.cs:12]: The same here.

**Review by Aiddam — APPROVED** [review]

**Pull request merged**